test/functional/local_transaction_controller_test.rb

```diff
@@ -137,6 +140,52 @@ class LocalTransactionControllerTest < ActionController::TestCase
         end
       end
 
+      context "for a Scotland local authority with devolved administration service availability" do
+        setup do
+          stub_mapit_has_a_postcode_and_areas(
+            "EH8 8DX",
+            [0, 0],
+            [
+              {
+                "name" => "Holyroodhouse",
+                "type" => "LGD",
+                "govuk_slug" => "edinburgh",
+                "country_name" => "Scotland",
+              },
+            ],
+          )
+
+          post :search, params: { slug: "send-a-bear-to-your-local-council", postcode: "EH8 8DX" }
+        end
+
+        should "sanitize postcodes and redirect to the slug for the appropriate authority tier" do
```

Comment:
I don't think this should is quite right for the scenario - particularly as the postcode isn't malformed. We should rephrase this more focused on the scenario we're testing.

Reply:
I changed it to `should "redirect to the local authority slug"`